### PR TITLE
Update ImplicitRemoteValidationIT.groovy

### PR DESCRIPTION
**Pull request by arvindkrishnakumar-okta**

### CODE REVIEW

File: src/main/groovy/com/okta/test/mock/tests/ImplicitRemoteValidationIT.groovy [hunk 1]
```diff
@@ -76,6 +76,6 @@ class ImplicitRemoteValidationIT extends ApplicationTestRunner {
         .then()
             .statusCode(401)
             .header("WWW-Authenticate", startsWith("Bearer"))
-            .body(Matchers.equalTo(ERROR_401))
+            .body(containsString("401 Unauthorized"))
```
Comment (bdemers) [inline src/main/groovy/com/okta/test/mock/tests/ImplicitRemoteValidationIT.groovy:79]: this body check _might_ not be valid technically a 401 doesn't need to return a body.
Our Spring Boot starter, has a utility method `Okta.configureResourceServer401ResponseBody(...)` to add a non-empty body. For developers testing resource server endpoints in a browser this is helpful.

TL;DR, I have no objection to this change, but consider just removing that line.

**Review by bdemers — APPROVED** [review]

File: src/main/groovy/com/okta/test/mock/tests/ImplicitRemoteValidationIT.groovy (hunk 1 above)
Comment (arvindkrishnakumar-okta) [reply src/main/groovy/com/okta/test/mock/tests/ImplicitRemoteValidationIT.groovy:79]: sure, removed that line.
